index.js

```diff
@@ -0,0 +1,40 @@
+import cors from 'cors';
+import express from 'express';
+import { pool } from './db.js';
+const PORT = process.env.PORT || 9000;
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+app.get('/', async function (req, res) {
+  const cards = await pool.query('SELECT * FROM cards');
+  res.send(cards.rows);
+});
+
+app.post('/create', async (req, res) => {
+  const { newNumber, cardId } = req.body;
+  const cards = await pool.query(`INSERT INTO cards VALUES
```

Comment:
what value is on this cards variable? If it is already containing what we need why you are getting all cards again in `if` statement?